redis/broker.go

```diff
@@ -68,10 +70,10 @@ type Broker struct {
 	pool           *redis.Pool
 	queues         []string
 	receiveTimeout int
+	mu             *sync.Mutex
```

Comment:
nit: i'd call the variable `mutex`

Reply:
its common practice. Event go creators call it mu https://github.com/golang/go/blob/e39c7a37f0f1a992eb65202d05c3148524c5c0ef/src/archive/zip/register.go#L41